packages/suite-desktop-core/src/modules/trezor-connect.ts

```diff
@@ -104,6 +104,10 @@ export const initBackground: ModuleInitBackground = ({ mainThreadEmitter, store
                 if (method === 'firmwareUpdate') {
                     const powerSaveBlocker = new PowerSaveBlocker();
                     powerSaveBlocker.startBlockingPowerSave();
+                    // todo: should or should not firmware update also block bioAuth from locking screen?
+                    // - when installing over bluetooth, it easily blocks the screen because it takes long
+                    // - on the other hand, it only visually locks screen and thats all, there is no cryptography or anything interesting happening
+                    //   so maybe it is actually good to have it locked
```

Comment:
what do you guys think? @komret @hynek-jina @vojtatranta

Reply:
More of like a product question.

But as Hynek says, since it basically only hides the UI and doesn't do any fancy encryption stuff, I would not mind it.

Altought, it would be pretty nice to have a mechanism there which would blocking in some cases. But that would be up for situation with true encryption.